### ISSUE TITLE
Sample for Elastic Search intergration

### ISSUE THREAD

**Issue by wonsuc** [firebase/functions-samples#165]
Though [Full-text search via Algolia](https://github.com/firebase/functions-samples/tree/master/fulltext-search) is great example for advanced search options on Firebase, many people still prefer Elastic Search rather than Algolia.

Therefore it would be really helpful if there was a sample like "Full-text search via Elastic Search".

I hope you consider it.
Thank you.

**Comment by vr001** [comment]
I agree, but its a feature request. Not sure, whether it needs to be under issues !!

**Comment by yasir-netlinks** [comment]
@wonsuc Hi, have you found a sample ? can we use firebase/flashlight for this ?

**Comment by wonsuc** [comment]
@yasir-netlinks You don't have to use `Flashlight` if you already have `Cloud Functions` server. If you want to use standalone `Node.js` server `Flashlight` would be the answer.
Though `Elastic Search` is nice and free, our team chose `Algolia`. It costs quite a bit but It's super fast and we saved lots of time for extra coding. `Elastic Search` is cool and full customizable, but you would need more `Back-end knowledge` and extra times for detail integrations.
Start `Elastic Search` with [GCP](https://cloudplatform.googleblog.com/2015/01/Deploy-Elasticsearch-on-Google-Compute-Engine.html), it's my best recommendation.
